### ISSUE TITLE
generate dataset iterator prefetch flag from Blueoil prefetch flag

### ISSUE THREAD

**Issue by joelN123** [blue-oil/blueoil#612]
## tl;dr :
A straightforward fix is to include the prefetch flag in the _blueoil_to_lmnet function of blueoil/generate_lmnet_config.py

## detailed explanation :

For the DatasetIterator, there is a flag called enable_prefetch, which specifies whether to use prefetch or not. The value fed to it is a flag called prefetch, from the LMNet config. The LMNet config is generated from the Blueoil config by the _blueoil_to_lmnet function in the file blueoil/generate_lmnet_config.py

However, the _blueoil_to_lmnet function does not mention prefetch. When running python blueoil/cmd/main.py train, I also find the generated config.py file does not contain a value for the prefetch flag.

There exists a corresponding flag in the Blueoil config, called dataset_prefetch, inside of common. But, it looks like this flag is not used to generate a prefetch flag for the LMNet config.

So, I think this creates two problems : 1. prefetch defaults to not being used, but most users would probably prefer to have prefetch enabled. 2. the value of dataset_prefetch flag in the Blueoil config yml file does not reflect what is done in training.

**Comment by Joeper214** [comment]
Thanks for reporting this bug. I was able to reproduce this and I'll be fixing this.
